helm/flink-kubernetes-operator/templates/rbac.yaml

```diff
@@ -61,26 +59,20 @@ rules:
       - update
       - patch
       - delete
+      - deletecollection
   - apiGroups:
-      - extensions
+      - apps
     resources:
-      - deployments
-      - ingresses
+      - deployments/scale
     verbs:
       - get
-      - list
-      - watch
-      - create
       - update
       - patch
-      - delete
   - apiGroups:
       - flink.apache.org
     resources:
       - flinkdeployments
-      - flinkdeployments/finalizers
```

Comment:
Won't removing finalizers here affect the fix provided in https://github.com/apache/flink-kubernetes-operator/pull/600/? The finalizers were added here specifically for OpenShift, I think it's related to the `OwnerReferencesPermissionEnforcement` admission plugin.

Reply:
I'm not the best to judge this. But at least from my point of view the "/finializers" resources are also making trouble with the deployment because it looks like the resources do not exist and (like mentioned above) Kyverno is blocking our deployment because it cannot add verbs to non existing resources. That's at least how I understand it.
Here is the output of available resources on our cluster for the flink operator:

```
API               Resource                 Verbs
flink.apache.org  flinkdeployments         delete,deletecollection,get,list,patch,create,update,watch
flink.apache.org  flinkdeployments/status  get,patch,update
flink.apache.org  flinkdeployments/scale   get,patch,update
flink.apache.org  flinksessionjobs         delete,deletecollection,get,list,patch,create,update,watch
flink.apache.org  flinksessionjobs/status  get,patch,update
```